dask-gateway-server/dask-gateway-proxy/cmd/dask-gateway-proxy/main.go

```diff
@@ -541,6 +545,7 @@ func (p *Proxy) runTCP(ln net.Listener, forwarder *Forwarder) {
 	}
 }
 
+//TODO: make this a skinny main?
```

Comment:
Could you reword this todo a bit so I can understand what you mean? Now I understand it only as "refactor to include less logic", but I'm not sure on any detail or the benefits are. I would agree there are benefits to doing so if that is a common practice, because that makes the code easier to navigate for experienced golang developers for example.

Reply:
You are correct that it is a common pattern of including less logic in the main function, though I think I'll get rid of the comment. With the scope of this PR and the current size of the dask-gateway-proxy go code, it wouldn't make it more beneficial to refactor `main` at this time.